src/main/java/core/basesyntax/WorkWithFile.java

```
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.util.Arrays;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] data = readCsv(fromFileName);
+        String[] sortedData = calculateData(data);
+        writeCsv(toFileName, sortedData);
+    }
+
+    private String[] readCsv(String fileName) {
+        File file = new File(fileName);
+        StringBuilder stringBuilder = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(file))) {
+            String line = bufferedReader.readLine();
+            while (line != null) {
+                stringBuilder.append(line).append(" ");
+                line = bufferedReader.readLine();
+            }
+        } catch (Exception e) {
+            throw new RuntimeException("Can't read file", e);
+        }
```

[Don’t ignore exceptions](https://mate-academy.github.io/jv-program-common-mistakes/java-core/builder-file/work-with-file#dont-ignore-exceptions)

---

Use more lower-level exception.